adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala

```diff
@@ -148,7 +148,9 @@ object ReferenceRegion {
    * @return The site where this genotype lives.
    */
   def apply(genotype: Genotype): ReferenceRegion = {
-    ReferenceRegion(genotype.getVariant)
+    ReferenceRegion(genotype.getContigName,
```

Comment:
A few general questions (not about this line specifically):

Rename `orientation` to `strand`?
Add strand parameters to other methods (e.g. `fromStart`, `toEnd`, `all`)?
Set strand when copying from one `ReferenceRegion` to another (i.e. line 234)?